### PR TITLE
add loginNetWeaver function

### PR DESCRIPTION
**Pull request by Templada**

### CODE REVIEW

File: src/reuse/modules/ui5/session.ts [hunk 1]
```diff
@@ -132,6 +132,25 @@ export class Session {
...
+  /**
+ * @function loginNetWeaver
+ * @memberOf ui5.session
+ * @description Login with sap netweaver for WebGUI form and specific username and password.
+ * @param {String} username - The username.
+ * @param {String} [password] - The password.
+ * @param {Boolean} [pressContinue=true] - Specifies if the function will press continue if applicable.
+ * @example await ui5.session.loginNetWeaver("john", "abc123!");
+ */
+  async loginNetWeaver(username: string, password?: string, pressContinue = true) {
```
Comment (Templada) [inline src/reuse/modules/ui5/session.ts:144]: thinking of moving it into nonUi5... wdyt? @DIFSRIP

Comment (DIFSRIP) [reply src/reuse/modules/ui5/session.ts:144]: I would move it to common maybe? Maybe the other logins as well at some point of time?

**Review by DIFSRIP — COMMENTED** [review]
looks good to me, maybe we can ad a short test for it. Just for local execution and / or a static one as for the others.

**Comment by Templada** [comment]
> looks good to me, maybe we can ad a short test for it. Just for local execution and / or a static one as for the others.

fine with me, then i just move the loginNetWeaver to common and create a new session module

**Review by DIFSRIP — APPROVED** [review]

**Pull request merged**